classes/db/DbPDO.php

```diff
@@ -240,6 +240,8 @@ public function Insert_ID()
      *
      * @see DbCore::Affected_Rows()
      *
+     * @throws Error if previous query failed silently (`$this->result` is not a `PDOStatement`)
+     *
```

Comment:
It would be best to throw an Exception if `$this->result` is not what is expected